### PR TITLE
Auto-assign blog PRs to kbarnard10

### PR DESCRIPTION
**Pull request by zacharysarah**
This PR:
- Adds @kbarnard10 as a reviewer for content/en/blog
- Adds @alexcontini as a reviewer for content/en/case-studies

These additions make them the sole reviewers at their respective levels of OWNERS permissions, which should hopefully auto-assign any relevant PRs to them and them only for blog post and case studies PRs respectively.

### CODE REVIEW

**Comment by kbarnard10** [comment]
@zacharysarah What happens, for example, if I submit a PR to the blog repo? Or if Alex submits a PR to the case study repo?

**Comment by zacharysarah** [comment]
@kbarnard10 You’ll have to get another approver to `/lgtm` and `/approve` your PR, same as usual, probably with an `/assign` in a comment or by reaching out on Gchat. 🙂

File: content/en/blog/OWNERS [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Owned by Kubernetes blog reviewers.
 options:
   no_parent_owners: true
-# reviewers: // List any reviewers here
+# reviewers:
```
Comment (mistyhacks) [inline content/en/blog/OWNERS]: I think this line (and the same line in the next file) will be ignored because of the leading comment characters?

Comment (zacharysarah) [reply content/en/blog/OWNERS]: @mistyhacks Fixed.

**Pull request merged**